Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoDecoderH264.mm

```diff
@@ -87,13 +116,16 @@ @implementation RTCVideoDecoderH264 {
   RTCVideoDecoderCallback _callback;
   OSStatus _error;
   bool _useAVC;
+  std::deque<std::unique_ptr<RTCVideoFrameWithOrder>> _reorderQueue;
```

Comment:
for safety, I would define on which thread _reorderQueue can be accessed (through `WTF_GUARDED_BY_CAPABILITY`) because at it looks, it seems to be racy

Reply:
We cannot really do that in flush. I guess we could introduce a queue for that specific purpose so that the decoder client could tell where to call callback.

Reply:
isn't there a contract on which thread callback will be called on ?